libs/components/lookup/src/lib/modules/lookup/lookup-autocomplete-adapter.ts

```diff
@@ -30,25 +30,33 @@ export class SkyLookupAutocompleteAdapter {
    * @default "name"
    */
   @Input()
-  public set descriptorProperty(value: string) {
-    this.#_descriptorProperty = value;
+  public set descriptorProperty(value: string | undefined) {
+    this.#_descriptorProperty = value ?? 'name';
   }
 
   public get descriptorProperty(): string {
-    return this.#_descriptorProperty || 'name';
+    return this.#_descriptorProperty;
   }
 
   /**
    * Specifies an array of object properties to search.
    * @default ["name"]
    */
   @Input()
-  public set propertiesToSearch(value: string[]) {
-    this.#_propertiesToSearch = value;
+  public set propertiesToSearch(value: string[] | undefined) {
+    this.#_propertiesToSearch = value ?? ['name'];
+
+    // Reset default search if it is what is being used.
+    if (this.search !== this.searchOrDefault) {
```

Comment:
Most of the same guidance from the autocomplete comment applies here as the code is very similar.